### ISSUE TITLE
The result of the first Toba Congress request to modify the "ba" character.

### ISSUE THREAD

**Issue by bennylin** [notofonts/batak#9]
**Is your feature request related to a problem? Please describe.**
News: https://www.ipminews.co.id/rumusan-hari-ketiga-kongres-i-kebudayaan-batak-toba-pada-22-oktober-2022/

One of the resolution of the congress in October 2022 was that the shape of "ba" should be modified into more elongated form, just like commonly seen in the Toba manuscripts for example "Pustaha laklak"
![image](https://github.com/notofonts/batak/assets/3002108/4ee2e96e-58ec-4cb1-b80f-7c0c1695b5e6)

This request is on behalf of Surung Simanullang.

**Describe the solution you'd like**
1BC5 ᯅ BATAK LETTER BA should be modified to become more elongated (see example in the above image). 
either the glyph is redrawn, or an alternative glyph is provided

**Describe alternatives you've considered**
Using its current form ᯅ is considered incorrect based on the forms found in the manuscripts

**Additional context**

![image](https://github.com/notofonts/batak/assets/3002108/39a2bc5a-6440-4ef8-a682-8fadc21792bc)
Add MSS 156789 p 7: "balang do" ᯅᯞᯰ ᯑᯬ, the letter "ba"/ᯅ is more rounded and other manuscripts

**Comment by simoncozens** [comment]
This is definitely a unification issue. Toba ba should be an oval, but other orthographies can keep the existing shape.

**Comment by simoncozens** [comment]
I'm tempted to say "just use U+1BC6 KARO BA", at least for the meantime. To make this work we either need a stylistic set which substitutes ba for karo ba, or we need an OpenType language tag for Karo, which we don't currently have.

**Comment by ulikozok** [comment]
Exactly. Use Karo BA and you're good. Also, 1BC5 ᯅ BATAK LETTER BA is not a "rare" form, but quite common.
